console/src/components/Common/utils/localStorageUtils.js

```diff
@@ -0,0 +1,12 @@
+export const getLocalStorageItem = key => {
```

Comment:
We also have utility functions for local storage here: https://github.com/hasura/graphql-engine/blob/master/console/src/utils/localstorage.js,
and they are not used anywhere. Maybe we can get rid of them, so there aren't multiple local storage utils files?

Reply:
@beerose That is true. This one seems to be a better implemementation of the functions but seems to be unused anywhere in the codebase as of now. Can you approve this for now. I'll raise another PR right after this with a refactor of the localStorage utils usage across the code.

Reply:
Ok. Cool.